pitch.md

```diff
@@ -57,13 +186,41 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+So I think it will be around 50/50 right now. If it was easy to allow users to 
+do these complex calendar designs then other calendar creation tools would
+have implemented them by now. It won't be easy to allow them a lot of creativity
+with their design without making it also harder to intially understand. 
+
+On the other hand, the output of the program won't be easy either. While it is 
+static, I have to compute reptitious events that might overlap with one another
+and return it in a general calendar file format. 
```

Comment:
How hard this part is will probably depend on how tricky the icalendar format is, and how well (if at all) the language you're working with supports the format.